scripts/list-build-dependencies.sh

```diff
@@ -99,23 +99,26 @@ function list_packages() {
 	case "$1" in
 
 		apt)
+			# Package repo: https://packages.ubuntu.com/
 			# Apt separates GCC into the gcc and g++ pacakges, the latter which depends on the prior.
 			# Therefore, we provide g++ in-place of gcc.
 			VERSION_DELIM="-"
 			if [[ "${COMPILER}" == "gcc" ]]; then
 				COMPILER="g++"
 			fi
-			PACKAGES=(libtool build-essential autoconf-archive libsdl1.2-dev libsdl-net1.2-dev libopusfile-dev libspeexdsp-dev)
+			PACKAGES=(libubsan[0-9] libasan[0-9] xvfb libtool build-essential autoconf-archive libsdl1.2-dev libsdl-net1.2-dev libopusfile-dev libspeexdsp-dev)
```

Comment:
I thought about it a bit more… I'm not sure if these new, sanitizer-related dependencies belong in here… They will slow down all Linux builds and make this script not useful for packaging in the future…

Let's move these dependencies straight into an .yml file, only in relevant jobs (just like Python packages are limited only to Clang static analyzer job).